src/ontology/ro-edit.owl

```diff
@@ -6789,6 +6792,37 @@ AnnotationAssertion(oboInOwl:created_by obo:RO_0019501 <https://orcid.org/0000-0
 AnnotationAssertion(rdfs:label obo:RO_0019501 "confers susceptibility to condition")
 SubObjectPropertyOf(obo:RO_0019501 obo:RO_0003304)
 
+# Object Property: obo:RO_0020101 (vessel supplies blood to)
+
+AnnotationAssertion(obo:IAO_0000115 obo:RO_0020101 "x vessel supplies blood to y if and only if x is a vessel that supplies blood directly or indirectly to an anatomical structure y.")
+AnnotationAssertion(terms:contributor obo:RO_0020101 <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(terms:date obo:RO_0020101 "2024-02-01T10:09:22Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label obo:RO_0020101 "vessel supplies blood to")
+SubObjectPropertyOf(obo:RO_0020101 obo:RO_0002178)
+ObjectPropertyDomain(obo:RO_0020101 obo:UBERON_0001981)
+ObjectPropertyRange(obo:RO_0020101 obo:UBERON_0000061)
+
+# Object Property: obo:RO_0020102 (vessel drains blood from)
+
+AnnotationAssertion(obo:IAO_0000115 obo:RO_0020102 "x vessel drains blood from y if and only if x is a vessel that drains blood directly or indirectly from an anatomical structure y.")
+AnnotationAssertion(terms:contributor obo:RO_0020102 <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(terms:date obo:RO_0020102 "2024-02-01T10:09:41Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label obo:RO_0020102 "vessel drains blood from")
+SubObjectPropertyOf(obo:RO_0020102 obo:RO_0002179)
+ObjectPropertyDomain(obo:RO_0020102 obo:UBERON_0001981)
+ObjectPropertyRange(obo:RO_0020102 obo:UBERON_0000061)
+
+# Object Property: obo:RO_0020103 (vessel directly supplies blood to and drains blood from)
+
+AnnotationAssertion(obo:IAO_0000115 obo:RO_0020103 "x vessel directly supplies blood to and drains blood from y if and only if x is a blood vessel, and it directly supplies blood to y and directly drains blood from y.")
```

Comment:
For consistency, maybe say that y is an anatomical entity?

Reply:
Done, thank you